### ISSUE TITLE
No way to turn off bluetooth

### ISSUE THREAD

**Issue by popohl** [friedrichweise/wechsel#19]
Hi!
Unless I haven't found it yet, there is no way to turn off bluetooth from the Wechsel interface.
It would be nice to implement that feature!

**Comment by friedrichweise** [comment]
You are right. This option is currently not implemented. I'll add it to the roadmap.
